clang/test/CIR/CodeGen/struct.cpp

```diff
@@ -12,6 +12,17 @@ IncompleteS *p;
 // LLVM: @p = dso_local global ptr null
 // OGCG: @p = global ptr null, align 8
 
+struct CompleteS {
```

Comment:
Do simple union work with this? If so, can you add a test for those, too?

Reply:
Yes, I think unions should work. I'll add a test for that.